agent/acs/handler/attach_task_eni_handler_test.go

```diff
@@ -323,7 +323,7 @@ func TestENIAckHappyPath(t *testing.T) {
 	manager := mock_statemanager.NewMockStateManager(ctrl)
 
 	mockWSClient := mock_wsclient.NewMockClientServer(ctrl)
-	eniAttachHandler := newAttachENIHandler(ctx, clusterName, containerInstanceArn, mockWSClient, taskEngineState, manager)
+	eniAttachHandler := newAttachTaskENIHandler(ctx, clusterName, containerInstanceArn, mockWSClient, taskEngineState, manager)
```

Comment:
question: so for the tests you removed from this file - it seem like you're covering all the cases with the instance type eni yea?

Reply:
yes